Makefile

```diff
@@ -1,13 +1,15 @@
 .PHONY: schema test clippy build
 
+TOOLCHAIN := "1.55.0"
```

Comment:
could you please explain (briefly) why we use 1.55.0 here but 1.58.1 in cosmos contracts?
https://github.com/lidofinance/lido-cosmos-hub-contracts/blob/main/Makefile

Reply:
Cosmos contracts repo is in an active development so we wanna use the latest version of rust.
This repo (`main` branch particularly) is deployed so we've freezed rust version here.
But we have a [develop](https://github.com/lidofinance/lido-terra-contracts/tree/develop) branch where we bumps rust version to the latest (not currently, but we'll do it with new features or before deployment definitely).